webpack/assets/javascripts/foreman_tools.js

```diff
@@ -80,3 +81,22 @@ export function initTypeAheadSelect(input) {
     width: '400px'
   });
 }
+
+// send get forms through turbolinks
+export function tableUpdated(element) {
+  let uri = new URI(window.location.href);
+  let search;
+
+  if (element !== undefined) {
+    search = $(element).find('.autocomplete-input');
+  } else {
+    search = $('#search');
+  }
+
+  if (search[0] !== undefined) {
```

Comment:
This is maybe premature, but the idea behind this function is to be called when table related data has changed, including per_page pagination change. in rare cases where there is no search for a given table that can happen.